cuML/CMakeLists.txt

```diff
@@ -71,8 +70,9 @@ set(CMAKE_IGNORE_PATH "${CMAKE_INSTALL_DIR}/lib" CACHE STRING
 set(GPU_ARCHS "" CACHE STRING
   "List of GPU architectures (semicolon-separated) to be compiled for")
 option(LINEINFO "Enable lineinfo in nvcc" OFF)
+option(MEMCHECK "Enable cudamemcheck function symbol backtrace in nvcc" ON)
 option(KERNELINFO "Enable kernel resource usage info" OFF)
-option(DEBUG "Get a debug build" OFF)
+option(DEBUG "Get a debug build" ON)
```

Comment:
Debug build should be OFF by default

Reply:
Yes it is for development I will turn off this flag